test/CMakeLists.txt

```diff
@@ -33,10 +34,18 @@ add_library (gtest STATIC IMPORTED)
 add_dependencies(gtest gtest_main)
 set_target_properties(gtest PROPERTIES IMPORTED_LOCATION "${PROJECT_BINARY_DIR}/lib/libgtest.a")
 
+# http://stackoverflow.com/a/36341753 - add gtest as dependency to all tests
```

Comment:
Can you add a `TODO(marehr): Once we require cmake>=3.7 use BUILDSYSTEM_TARGETS instead`